<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
<title>packages/plotutils/plotutils-info.patch - view - 1.2</title>
<meta name="robots" content="nofollow" />
<meta name="generator" content="FreeBSD-CVSweb 3.0.6" />
<meta http-equiv="Content-Script-Type" content="text/javascript" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<link rel="stylesheet" type="text/css" href="/cvsweb/css/cvsweb.css" />
</head>
<body class="src">
<table class="navigate-header" width="100%" summary="Navigation">
 <tr>
  <td>
<a href="./plotutils-info.patch#rev1.2"><img src="/cvsweb/icons/back.gif" alt="[BACK]" border="0" width="20" height="22" /></a><b>Return to <a href="./plotutils-info.patch#rev1.2">plotutils-info.patch</a> CVS log</b> <img src="/cvsweb/icons/text.gif" alt="[TXT]" border="0" width="20" height="22" /></td>  <td style="text-align: right"><img src="/cvsweb/icons/dir.gif" alt="[DIR]" border="0" width="20" height="22" /> <b>Up to  <a href="/cgi-bin/cvsweb.cgi/#dirlist">[Local Repository]</a> / <a href="/cgi-bin/cvsweb.cgi/packages/#dirlist">packages</a> / <a href="/cgi-bin/cvsweb.cgi/packages/plotutils/#dirlist">plotutils</a></b></td>
 </tr>
</table>
<hr />
<div class="log-markup">
File:&nbsp;
 <a href="/cgi-bin/cvsweb.cgi/#dirlist">[Local Repository]</a> / <a href="/cgi-bin/cvsweb.cgi/packages/#dirlist">packages</a> / <a href="/cgi-bin/cvsweb.cgi/packages/plotutils/#dirlist">plotutils</a> / <a href="/cgi-bin/cvsweb.cgi/packages/plotutils/plotutils-info.patch">plotutils-info.patch</a><br />
<a name="rev1.2"></a><a name="auto-ti-plotutils-2_5_1-4"></a><a name="auto-ti-plotutils-2_5_1-3"></a><a name="auto-ti-plotutils-1_3-2"></a><a name="auto-th-plotutils-2_5_1-4"></a><a name="auto-th-plotutils-2_5_1-3"></a><a name="auto-th-plotutils-2_5-1"></a><a name="auto-th-plotutils-1_3-2"></a><a name="HEAD"></a><a name="MAIN"></a>
 Revision <b>1.2</b>: <a href="/cgi-bin/cvsweb.cgi/~checkout~/packages/plotutils/plotutils-info.patch?rev=1.2;content-type=text%2Fplain" class="download-link">download</a> - view: <a href="plotutils-info.patch?rev=1.2;content-type=text%2Fplain" class="display-link">text</a>, <a href="plotutils-info.patch?annotate=1.2">annotated</a> - <a href="plotutils-info.patch?r1=1.2#rev1.2">select&nbsp;for&nbsp;diffs</a> - <a href="?graph=1">revision graph</a><br />
<i>Sun May 14 19:22:14 2006 UTC</i> (2 years, 11 months ago) by <i>qboosh</i><br />
Branches: <a href="./plotutils-info.patch?only_with_tag=MAIN">MAIN</a><br />
CVS tags: <a href="./plotutils-info.patch?only_with_tag=auto-ti-plotutils-2_5_1-4">auto-ti-plotutils-2_5_1-4</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-ti-plotutils-2_5_1-3">auto-ti-plotutils-2_5_1-3</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-ti-plotutils-1_3-2">auto-ti-plotutils-1_3-2</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-th-plotutils-2_5_1-4">auto-th-plotutils-2_5_1-4</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-th-plotutils-2_5_1-3">auto-th-plotutils-2_5_1-3</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-th-plotutils-2_5-1">auto-th-plotutils-2_5-1</a>,
<a href="./plotutils-info.patch?only_with_tag=auto-th-plotutils-1_3-2">auto-th-plotutils-1_3-2</a>,
<a href="./plotutils-info.patch?only_with_tag=HEAD">HEAD</a><br />
<pre class="log">
- updated for 2.5
</pre>
</div>
<hr /><pre>
--- plotutils-2.4.1/info/plotutils.texi	Mon Jul  3 07:09:05 2000
+++ plotutils-2.4.1.new/info/plotutils.texi	Wed Feb 14 04:24:31 2001
@@ -6,18 +6,18 @@
 @c @setchapternewpage odd
 @c %**end of header
 
-@dircategory GNU Plotting Utilities
+@dircategory Scientific software:
 @direntry
-* Plotting utilities: (plotutils).      GNU plotting utilities.
-* graph: (plotutils)graph Invocation.   Plot datasets, possibly in real time.
-* plot: (plotutils)plot Invocation.     Convert and display plot files.
-* pic2plot: (plotutils)pic2plot Invocation.   Convert files in the pic language
-* tek2plot: (plotutils)tek2plot Invocation.   Translate legacy Tektronix data.
-* plotfont: (plotutils)plotfont Invocation.   Plot character maps of fonts.
-* spline: (plotutils)spline Invocation. Interpolate between points in datasets.
-* ode: (plotutils)ode Invocation.       Integrate differential equations.
-* libplot: (plotutils)libplot.          A library for 2-D vector graphics.
-* Appendices: (plotutils)Appendices.    More info on the plotting utilities.
+* Plotting utilities: (plotutils).	GNU plotting utilities
+* graph: (plotutils)graph.		Plot datasets, possibly in real time
+* plot: (plotutils)plot.		Convert and display plot files
+* pic2plot: (plotutils)pic2plot.	Convert files in the pic language
+* tek2plot: (plotutils)tek2plot.	Translate legacy Tektronix data
+* plotfont: (plotutils)plotfont.	Plot character maps of fonts
+* spline: (plotutils)spline.		Interpolate between points in datasets
+* ode: (plotutils)ode.			Integrate differential equations
+* libplot: (plotutils)libplot.		A library for 2-D vector graphics
+* Appendices: (plotutils)Appendices.	More info on the plotting utilities
 @end direntry
 
 @iftex
--- plotutils-2.4.1/libxmi/info/libxmi.texi	Sun Nov 21 21:46:24 1999
+++ plotutils-2.4.1.new/libxmi/info/libxmi.texi	Wed Feb 14 04:35:10 2001
@@ -6,9 +6,10 @@
 @c @setchapternewpage odd
 @c %**end of header
 
-@dircategory Libraries
+@dircategory Libraries:
 @direntry
-* Libxmi: (libxmi).           The GNU libxmi 2-D rasterization library.
+* Libxmi: (libxmi).			The GNU libxmi 2-D
+					rasterization library
 @end direntry
 
 @ifinfo
</pre>
<hr />
<address><span style="font-size: smaller">PLD-CVSweb &lt;<a href="mailto:feedback@pld-linux.org">feedback@pld-linux.org</a>&gt;</span></address>
</body>
</html>
